test/metabase/email_test.clj

```diff
@@ -359,3 +361,79 @@
               (is (re-find
                    #"(?s)Content-Disposition: attachment.+filename=.+this-is-quite-[\-\s?=0-9a-zA-Z]+-characters.csv"
                    (m/mapply email/send-message! params-with-problematic-file))))))))))
+
+(deftest throttle-test
+  (let [send-email (fn [recipients]
+                     (with-redefs [postal/send-message (fn [& args] (last args))]
+                       (email/send-email!
+                        {}
+                        (merge {:from    "awesome@metabase.com"
+                                :subject "101 Reasons to use Metabase"
+                                :body    "101. Metabase will make you a better person"}
+                               recipients))))]
+    (tu/with-temporary-setting-values
```

Comment:
Can we add some test case(s) for 0 recipients?

Reply:
if you mean that sending an email with 0 recipients is ok, then it's on line 380? or maybe we want another test for when we reached the limit yea?

Reply:
I missed it! But yeah the extra test you suggest sounds good too.